config-helper.js

```diff
@@ -50,6 +50,33 @@ module.exports = {
 			this.replaceMain({entry: config.entry[opts.chunk || 'main']}, replacement, opts);
 		}
 	},
+	replaceEntry: function (config, replacement, opts = {}) {
+		if (typeof replacement === 'string') {
+			try {
+				replacement = JSON.parse(replacement);
```

Comment:
I don't prefer mutating function arguments.
Would you please add another variable?

Reply:
OK